docs/kubernetes-installers.md

```diff
@@ -19,8 +19,8 @@ work with that Antrea version.
 | - | Kops v1.20, K8s v1.20.5 | AWS EC2 | Ubuntu 20.04.2 LTS (5.4.0-1041-aws) amd64, containerd://1.4.4 | t3.medium | [results tarball](http://downloads.antrea.io/artifacts/sonobuoy-conformance/kops_202104212218_sonobuoy_bf0f8e77-c9df-472a-85e2-65e456cf4d83.tar.gz) |  |
 | - | EKS, K8s v1.17.12 | AWS | AmazonLinux2, docker | t3.medium |  | Antrea CI |
 | - | GKE, K8s v1.19.8-gke.1600 | GCP | Ubuntu 18.04, docker | e2-standard-4 |  | Antrea CI |
-| - | AKS, K8s v1.18.14 | Azure | Ubuntu 18.04, moby | Standard_DS2_v2 |  | Antrea CI |
-| - | AKS, K8s v1.19.9 | Azure | Ubuntu 18.04, containerd | Standard_DS2_v2 |  | Antrea CI |
+| v2.4.0 | AKS, K8s v1.18.14 | Azure [^Disclaimer] | Ubuntu 18.04, moby | Standard_DS2_v2 |  | Antrea CI |
+| v2.4.0 | AKS, K8s v1.19.9 | Azure [^Disclaimer] | Ubuntu 18.04, containerd | Standard_DS2_v2 |  | Antrea CI |
```

Comment:
I am not sure this change is needed (is Ubuntu 18.04 correct?), but if you want to keep it, it should be moved to the end of the table

Reply:
Thanks for the comment. Ubuntu 18.04 is not accurate. Now Azure account was closed, we cannot check the last verified K8s and Ubuntu versions with Antrea v2.4.0. I removed the two rows.